aea/cli/launch.py

```diff
@@ -37,50 +38,51 @@ def _run_agent(click_context, agent_directory: str):
     click_context.invoke(run)
 
 
-@click.command()
-@click.argument("agents", nargs=-1, type=AgentDirectory())
-@pass_context
-def launch(click_context, agents: List[str]):
-    """Launch many agents."""
-    agents_directories = list(map(Path, list(OrderedDict.fromkeys(agents))))
-    agent_processes = [
-        Process(target=_run_agent, args=(click_context, agent_directory))
-        for agent_directory in agents_directories
-    ]
+def _launch_subprocesses(agents: List[Path]):
+    """
+    Launch many agents using subprocesses.
 
+    :param agents: list of paths to agent projects.
+    :return: None
+    """
+    processes = []
     failed = 0
+    for agent_directory in agents:
+        process = Popen(  # nosec
+            [sys.executable, "-m", "aea.cli", "run"], cwd=str(agent_directory)
```

Comment:
The verbosity option is not forwarded.
Also, how to handle the `--connections` option for every agent?

Reply:
Suggests default connection should be a list! Then it can be set in the config.